tests/fixtures/require-spread/good/super.js

```diff
@@ -4,6 +4,7 @@ let Blah = Foo.extend({
 
     // odd, but still needed in some cases
     let { _super } = this;
+
```

Comment:
It was to enforce this https://github.com/DockYard/ember-suave/tree/master/tests/fixtures/rules/require-line-break-after-variable-assignment

Do we want to no longer enforce that?